amo2kinto/synchronize.py

```diff
@@ -37,8 +61,10 @@ def push_changes(diff, kinto_client, bucket, collection):
             # Records are enabled by default.
             record['enabled'] = True
             batch.create_record(record)
+        for record in to_update:
+            batch.patch_record(strip_keys(record))
```

Comment:
Why do you strip keys ? (add comment in code) Especially if strip keys are id, last_modified and enabled, how does it know which record is to be patched ? what if enabled field is changed from false to true ? without last modified don't we if-match headers ?